packages/proxy/test/unit/http/util/service-worker-injector.spec.ts

```diff
@@ -0,0 +1,181 @@
+import { expect } from 'chai'
+import { injectIntoServiceWorker } from '../../../../lib/http/util/service-worker-injector'
+
+describe('lib/http/util/service-worker-injector', () => {
+  describe('injectIntoServiceWorker', () => {
+    it('injects into the service worker', () => {
+      const actual = injectIntoServiceWorker(Buffer.from('foo'))
+
+      const expected = `
+      let __cypressIsScriptEvaluated = false;
+      (function __cypressInjectIntoServiceWorker() {
```

Comment:
This string is going to change every time `service-worker-injector.ts#__cypressInjectIntoServiceWorker` changes - is there a way to make this less brittle?

Reply:
Updated to not verify the actual contents of the `__cypressInjectIntoServiceWorker` function. [798f4ae](https://github.com/cypress-io/cypress/pull/28900/commits/798f4ae11628b8f98f15248bc9888217e0c9a2d3)